### ISSUE TITLE
azurerm_function_app and azurerm_storage_account showing as not supported on Infracost 0.8.7

### ISSUE THREAD

**Issue by holzro** [infracost/infracost#757]
When I try to run a cost breakdown with azurerm_function_app and azurerm_storage_account resources, they show as not supported, even when the usage file is populated. This is on Infracost 0.8.7

```
❯ infracost breakdown --usage-file infracost-usage.yml --sync-usage-file --path ./ --show-skipped
Detected Terraform directory at ./
  ✔ Running terraform plan

✔ Calculating monthly cost estimate

Project: ./

 Name                                    Monthly Qty  Unit   Monthly Cost

 azurerm_app_service_plan.function
 └─ Instance usage (Y1)                            0  hours         $0.00

 PROJECT TOTAL                                                      $0.00

----------------------------------
3 resource types weren't estimated as they're not supported yet.
Please watch/star https://github.com/infracost/infracost as new resources are added regularly.
1 x azurerm_application_insights
1 x azurerm_function_app
1 x azurerm_storage_account
```

```
version: 0.1
resource_usage:
  azurerm_function_app.function:
    execution_duration_ms: 1000
    instances: 0
    memory_mb: 128
    monthly_executions: 1000000000
  azurerm_storage_account.function:
    blob_index_tags: 0
    data_at_rest_storage_gb: 1
    early_deletion_gb: 0
    metadata_at_rest_storage_gb: 1
    monthly_data_retrieval_gb: 0
    monthly_data_write_gb: 1
    monthly_list_and_create_container_operations: 0
    monthly_other_operations: 0
    monthly_read_operations: 0
    monthly_write_operations: 10000
    snapshots_storage_gb: 0
    storage_gb: 1
```

**Comment by alikhajeh1** [comment]
Can you possibly paste the `azurerm_function_app` and `azurerm_storage_account` TF resource blocks? That helps check if there's a condition in them that's causing us to skip them.

**Comment by holzro** [comment]
Here you go 

```
resource "azurerm_storage_account" "function" {
  name                      = var.storage_name
  resource_group_name       = azurerm_resource_group.rg.name
  location                  = azurerm_resource_group.rg.location
  account_tier              = "Standard"
  account_replication_type  = "LRS"
  enable_https_traffic_only = true
  min_tls_version           = "TLS1_2"
  tags                      = local.common_tags
}

resource "azurerm_function_app" "function" {
  name                       = var.fun_name
  location                   = azurerm_resource_group.rg.location
  resource_group_name        = azurerm_resource_group.rg.name
  app_service_plan_id        = azurerm_app_service_plan.function.id
  storage_account_name       = azurerm_storage_account.function.name
  storage_account_access_key = azurerm_storage_account.function.primary_access_key
  https_only                 = true
  version                    = "~3"
  tags                       = local.common_tags

  site_config {
    http2_enabled = true
    scm_type = "VSTSRM"
  }

  identity {
    type = "SystemAssigned"
  }

  lifecycle {
    ignore_changes = [app_settings["WEBSITE_RUN_FROM_PACKAGE"]]
  }
}
```

**Comment by alikhajeh1** [comment]
Awesome, can you possibly also add the TF resource block that `app_service_plan_id = azurerm_app_service_plan.function.id` is pointing to?
Is the `azurerm_app_service_plan.function` resource defined in the TF project that is being processed by Infracost, or is it in a separate module?

**Comment by alikhajeh1** [comment]
@holzro we pushed a fixed for this in v0.9.0 (released just now) - plz let us know if that works for you.

Re the `azurerm_storage_account` issue, plz run with `--log-level=warn` and create a new issue if you'd like us to dig into it.
